### PR TITLE
[browser] Cleanup domContentLoaded property and signal. JB#56173 OMP#JOLLA-504

### PR DESCRIPTION
**Pull request by rainemak**

### CODE REVIEW

**Comment by rainemak** [comment]
See also https://github.com/sailfishos/qtmozembed/pull/17

File: apps/qtmozembed/declarativewebpage.cpp [hunk 1]
```diff
@@ -203,15 +200,7 @@ void DeclarativeWebPage::restoreHistory() {
 
 void DeclarativeWebPage::setContentLoaded()
```
Comment (okodron) [inline apps/qtmozembed/declarativewebpage.cpp:201]: Can it be dropped as empty and unused?

Comment (mkenttala) [reply apps/qtmozembed/declarativewebpage.cpp:201]: Looks like it should be dropped, otherwise LGTM.

**Review by mkenttala — APPROVED** [review]

**Review by paveltkv — APPROVED** [review]

File: apps/qtmozembed/declarativewebpage.cpp (hunk 1 above)
Comment (rainemak) [reply apps/qtmozembed/declarativewebpage.cpp:201]: Will drop now.

Comment (rainemak) [reply apps/qtmozembed/declarativewebpage.cpp:201]: Oops. Accidentally left completely empty method there but hey that's why we have reviews.

Comment (rainemak) [reply apps/qtmozembed/declarativewebpage.cpp]: Fixed now.

**Comment by rainemak** [comment]
Rebased and removed empty DeclarativeWebPage::setContentLoaded

**Review by okodron — APPROVED** [review]

**Pull request merged**